backend/tests/test_graphql_api/test_user/test_query_current_user.py

```diff
@@ -228,6 +229,200 @@ def test_query_current_user__unit_admin(graphql):
     }
 
 
+@patch_method(HelsinkiProfileClient.ensure_token_valid)
+def test_query_current_user__unit_admin__unit_group(graphql):
+    unit = UnitFactory.create()
+    unit_group = UnitGroupFactory.create(units=[unit])
+
+    user = UserFactory.create_with_unit_role(unit_groups=[unit_group])
+    unit_role = user.unit_roles.first()
+    graphql.force_login(user)
+
+    fields = """
+        pk
+        unitRoles {
+            role
+            units {
+                pk
+            }
+            unitGroups {
+                pk
+            }
+            assigner {
+                firstName
+                lastName
+            }
+            createdAt
+            updatedAt
+            permissions
+        }
+    """
+
+    response = graphql(current_user_query(fields=fields))
+
+    assert response.has_errors is False, response
+    assert response.first_query_object == {
+        "pk": user.pk,
+        "unitRoles": [
+            {
+                "role": unit_role.role,
+                "units": [],
+                "unitGroups": [
+                    {
+                        "pk": unit_group.pk,
+                    },
+                ],
+                "assigner": None,
+                "createdAt": unit_role.created_at.isoformat(),
+                "updatedAt": unit_role.updated_at.isoformat(),
+                "permissions": [
+                    "CAN_CREATE_STAFF_RESERVATIONS",
+                    "CAN_MANAGE_APPLICATIONS",
+                    "CAN_MANAGE_NOTIFICATIONS",
+                    "CAN_MANAGE_RESERVATIONS",
+                    "CAN_MANAGE_RESERVATION_RELATED_DATA",
+                    "CAN_MANAGE_RESERVATION_UNITS",
+                    "CAN_VIEW_APPLICATIONS",
+                    "CAN_VIEW_RESERVATIONS",
+                    "CAN_VIEW_USERS",
+                ],
+            },
+        ],
+    }
+
+
+@patch_method(HelsinkiProfileClient.ensure_token_valid)
+def test_query_current_user__unit_admin__multiple_roles(graphql):
```

Comment:
Doesn't affect the content of the test, but strictly speaking, `__unit_staff__` rather than `__unit_admin__`